pyproject.toml

```
@@ -11,20 +11,24 @@ readme = "README.md"
 repository = "https://github.com/supabase-community/realtime-py"
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.8.18"
```

Consider moving to Python 3.9 as it looks like we might have [deprecated support for 3.8
](https://github.com/supabase/realtime-py/pull/182) while the PR was in progress

```suggestion
python = "^3.9.8"
```